package.json

```diff
@@ -2,7 +2,7 @@
   "name": "victory-core",
   "version": "4.4.1",
   "description": "Victory Core",
-  "main": "lib/index.js",
```

Comment:
@boygirl ohoh this is not going to change while merge, I change this is because I could easier for me test `victory-core` with `victory-chart` locally.